tsconfig.json

```diff
@@ -2,8 +2,11 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "React Native",
   "compilerOptions": {
+    "baseUrl": "./",
+    "paths": {
+        "@/*": ["src/*"]
+    },
     "isolatedModules": true,
-    "strict": true,
```

Comment:
In the future let's please keep our PRs focused on the specific feature we're coding against. Many of the changes in this PR are unnecessary when the task is to provide a UI for several screens.